os/board/rtl8730e/src/rtl8730e_boot.c

```diff
@@ -240,8 +240,10 @@ void board_gpio_initialize(void)
 		u32 pinpull;
 	} pins[] = {
 				{PA_23, PIN_INPUT, PullNone},
+				{PA_4, PIN_INPUT, PullNone},
```

Comment:
I can add the condition here #if CONFIG_RTL8730E_BOARD_REVISION >= 5, but that needs I2C1 to be disabled (I2C1 is not used). 

Hi @sunghan-chang ,
Do you think we should introduce other macro for such gpio wakeup pin? As either >=5 or <5 have peripheral uses PA_4, it might create unnecessary issue.